pkg/workflows/management/create_install_resources.go

```diff
@@ -0,0 +1,33 @@
+package management
+
+import (
+	"context"
+
+	"github.com/aws/eks-anywhere/pkg/logger"
+	"github.com/aws/eks-anywhere/pkg/task"
+	"github.com/aws/eks-anywhere/pkg/workflows"
+)
+
+type installResourcesOnManagementTask struct{}
```

Comment:
can we add a description to this one? it's not very clear what it does based on the name